style.css

```diff
@@ -1 +1,75 @@
 /*  Add your styling here */
+
+.flex-container {
+  display: flex;
+  background-color: #d3d3d3;
+  justify-content: space-around;
+}
+
+.flex-container > div {
+  background-color: #a9a9a9;
+  margin: 0.627em;
+  padding: 1.255em;
+  font-size: 1.882em;
+}
+
+.nav-bar {
+  display: flex;
+  background-color: #d3d3d3;
+  justify-content: space-around;
+  flex-direction: row-reverse;
+}
+
+.nav-bar > div {
+  background-color: #0a0101;
+  margin: 0.627em;
+  padding: 0.1em;
+  font-size: 1.882em;
+  color: white;
+  margin-top: 3.764em;
+}
+
+.main-content {
+  display: flex;
+  background-color: #a9a9a9;
+}
+
+.main-content > div {
+  background-color: #d3d3d3;
+  margin: 0.627em;
+  padding: 3.137em;
+  font-size: 1.875em;
+}
+
+.main-content div:nth-child(2) > p {
```

Comment:
In this case, it may have been better to add a new class, than target paragraphs using nth-child.